release_images.yml

```diff
@@ -352,3 +292,15 @@ release_images:
       example: True
       disable_sm_tag: False  # [Default: False] This option is not used by Example images
       force_release: False
+  22:
+    framework: "tensorflow"
+    version: "1.15.0"
+    inference:
+      device_types: ["eia"]
+      python_versions: ["py36"]
+      os_version: "ubuntu18.04"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+                            # to images being published.
+      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+                            # has already been published. Re-released image will have minor version incremented by 1.
```

Comment:
nit: Please add a `newline` at the end of the file, because this is a config file that needs to be read and parsed, which is dependent on the implementation of the module that parses this file.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Can be ignored for this PR in particular.